styles/ui-primitives.css

```
@@ -160,3 +168,129 @@ button.btn-icon:focus-visible {
+  cursor: not-allowed;
+}
+
+/* 視覺隱藏原生 checkbox，但保留可聚焦/可點擊與螢幕報讀器語意 */
+.switch-input {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  padding: 0;
+  border: 0;
+  clip: rect(0 0 0 0);
```

Stylelint (17.13.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)